packages/osl-operator-bundle-image/generated/bundle/manifests/logic-operator-rhel8-controllers-config_v1_configmap.yaml

```diff
@@ -28,36 +28,35 @@ data:
     # Default image used internally by the Operator Managed Kaniko builder to create the executor pods
     kanikoExecutorImageTag: gcr.io/kaniko-project/executor:v1.9.0
     # The Jobs Service image to use, if empty the operator will use the default Apache Community one based on the current operator's version
-    jobsServicePostgreSQLImageTag: "docker.io/apache/incubator-kie-kogito-jobs-service-postgresql:main"
-    jobsServiceEphemeralImageTag: "docker.io/apache/incubator-kie-kogito-jobs-service-ephemeral:main"
+    jobsServicePostgreSQLImageTag: "docker.io/apache/incubator-kie-kogito-jobs-service-postgresql:1.36.0"
+    jobsServiceEphemeralImageTag: "docker.io/apache/incubator-kie-kogito-jobs-service-ephemeral:1.36.0"
     # The Data Index image to use, if empty the operator will use the default Apache Community one based on the current operator's version
-    dataIndexPostgreSQLImageTag: "docker.io/apache/incubator-kie-kogito-data-index-postgresql:main"
-    dataIndexEphemeralImageTag: "docker.io/apache/incubator-kie-kogito-data-index-ephemeral:main"
+    dataIndexPostgreSQLImageTag: "docker.io/apache/incubator-kie-kogito-data-index-postgresql:1.36.0"
+    dataIndexEphemeralImageTag: "docker.io/apache/incubator-kie-kogito-data-index-ephemeral:1.36.0"
+    # The Kogito PostgreSQL DB Migrator image to use (TBD: to replace with apache image)
+    dbMigratorToolImageTag: "docker.io/apache/kie-kogito-db-migrator-tool:main"
```

Comment:
these should point to registry.redhat.io... and use productized name and tags

Reply:
I have sent suggestions with the expected names